src/Sulu/Bundle/CoreBundle/Build/CacheBuilder.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CoreBundle\Build;
+
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * @deprecated The cache clear builder will be remove with sulu 2.0 use the symfony commands instead.
+ *
+ * Builder for clearing the cache.
+ */
+class CacheBuilder extends SuluBuilder
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'cache';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDependencies()
+    {
+        return [];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function build()
+    {
+        @trigger_error(
+            'CacheBuilder is deprecated since version 1.6.10 and will be removed in 2.0 use the default symfony command cache:clear instead.',
+            E_USER_DEPRECATED
+        );
+
+        if (version_compare(Kernel::VERSION, '3.4.0') >= 0) {
+            @trigger_error(
+                'Skip clearing the cache with sulu:build as it is not longer supported with Symfony >=3.4 use cache:clear command instead.',
```

Comment:
Is there no way we can really log something to the terminal? I think we really have to make people aware, that this step is not doing anything.